app/src/test/java/ch/hikemate/app/authentication/AuthViewModelTest.kt

```diff
@@ -78,23 +99,33 @@ class AuthViewModelTest {
   fun signInWithGoogle_calls_repository_and_updates_currentUser_on_success() = runTest {
     setupSignedOutUser()
 
-    // Simulate a successful Google sign-in by invoking the onSuccess callback
-    doAnswer { arguments ->
-          val onSuccess = arguments.getArgument<(FirebaseUser?) -> Unit>(0)
-          onSuccess(mockFirebaseUser) // Call the success callback with a mock user
+    // Mock the sign-in process to immediately invoke the onSuccess callback with a mock user
+    doAnswer { invocation ->
+          val onSuccess = invocation.getArgument<(FirebaseUser?) -> Unit>(0)
+          onSuccess(mockFirebaseUser) // Trigger success callback with mock user
           null
         }
-        .`when`(mockRepository)
+        .whenever(mockRepository)
```

Comment:
Question: Is there a reason why you changed this ?

Reply:
There is no particular reason. I changed it back.
It is most likely when refactoring tests with copilot